File: spec/code_samples/PHP/payments@{id}@captures/post.php
<?php

$checkout = new CheckoutApi('your secret key');
$paymentID = 'pay_y3oqhf46pyzuxjbcn2giaqnb44';

// Full Capture
$refund = new Capture($paymentID);

// Or partial capture
$refund = new Capture($paymentID);
$refund->reference = 'your reference';
$refund->amount = 100;

return $checkout->payments()->capture($refund);

File: spec/code_samples/C#/payments@{id}@captures/post.cs
var api = CheckoutApi.Create("your secret key");
var paymentId = "pay_y3oqhf46pyzuxjbcn2giaqnb44";

// Full capture
await _api.Payments.CaptureAsync(paymentId);

// Or partial capture
var captureRequest = new CaptureRequest
{
    Reference = "your reference",
    Amount = 100
};

await _api.Payments.CaptureAsync(paymentId, captureRequest);
